### ISSUE TITLE
System response when a user registered with sign-up is pressed mistakenly

### ISSUE THREAD

**Issue by inrosh** [divizia/db-constructor#10]
The registered user enters the application without the right to access the content (HTTP error - 403) when entering the correct Username, Password fields in the form of authorization and clicking the Sign-up button 
![registered user click sign-up](https://user-images.githubusercontent.com/102860611/200806308-3c143ffd-d246-4a54-897b-063cac5080f3.png)

**Comment by Serhx4** [comment]
Hmm I wasn't able to repeat this bug.
I always get "Sign up" page

**Comment by inrosh** [comment]
Yes, I went in and went out several times and somewhere for the third time gives such a mistake

**Comment by Serhx4** [comment]
Ohh. You're very persistent. It's a very good quality for your profession :)
Now I should try it again
